packages/netlify-cms-widget-object/src/ObjectControl.js

```diff
@@ -46,16 +54,59 @@ export default class ObjectControl extends Component {
     return true;
   }
 
+  processControlRef = (field, wrappedControl) => {
+    if (!wrappedControl) return;
+    const name = field.get('name');
+    const list = field.get('widget') == 'list';
```

Comment:
Same as the first comment on `EditorControlPane.js`:

```suggestion
    const widget = field.get('widget');
    
    if (widget === 'list' || widget === 'object') {
      this.componentValidate[name] = wrappedControl.innerWrappedControl.validate;
    } else {
      this.componentValidate[name] = wrappedControl.validate;
    }
```